src/includes/class-health-check-wp-debug.php

```diff
@@ -0,0 +1,536 @@
+<?php
+
+/**
+ * Enables / disables WP_DEBUG.
+ *
+ * @package Health Check
+ */
+
+/**
+ * Class Health Check WP Debug
+ */
+class Health_Check_WP_Debug {
+
+	/**
+	 * Enables WP_DEBUG
+	 *
+	 * @uses copy()
+	 * @uses file()
+	 * @uses file_put_contents()
+	 * @uses fopen()
+	 * @uses strpos()
+	 * @uses fputs()
+	 * @uses fclose()
+	 * @uses wp_send_json_error()
+	 * @uses wp_send_json_succes()
+	 *
+	 * @return void
+	 */
+	static function enable_wp_debug() {
+
+		$wpconfig = ABSPATH . 'wp-config.php';
+
+		$wp_debug_found = 'no';
+
+		$editing_wpconfig = file( $wpconfig );
+
+		file_put_contents( $wpconfig, '' );
+
+		$write_wpconfig = fopen( $wpconfig, 'w' );
+
+		foreach ( $editing_wpconfig as $line ) {
+			if ( false !== strpos( $line, "'WP_DEBUG'" ) || false !== strpos( $line, '"WP_DEBUG"' ) ) {
+				$line           = "define( 'WP_DEBUG', true );" . PHP_EOL;
+				$wp_debug_found = 'yes';
+			}
+			fputs( $write_wpconfig, $line );
+		}
+
+		fclose( $write_wpconfig );
+
+		if ( 'no' === $wp_debug_found ) {
+
+			$editing_wpconfig = file( $wpconfig );
+
+			file_put_contents( $wpconfig, '' );
+
+			$write_wpconfig = fopen( $wpconfig, 'w' );
+
+			foreach ( $editing_wpconfig as $line ) {
+				if ( false !== strpos( $line, 'stop editing!' ) ) {
+					$line  = "define( 'WP_DEBUG', true );" . PHP_EOL;
+					$line .= "/* That's all, stop editing! Happy blogging. */" . PHP_EOL;
+				}
+				fputs( $write_wpconfig, $line );
+			}
+
+			fclose( $write_wpconfig );
+		}
+
+		$response = array(
+			'status'  => 'success',
+			'message' => esc_html__( 'WP_DEBUG was enabled.', 'health-check' ),
+		);
+
+		wp_send_json_success( $response );
+
+	}
+
+	/**
+	 * Disables WP_DEBUG
+	 *
+	 * @uses fopen()
+	 * @uses copy()
+	 * @uses wp_send_json_error()
+	 * @uses wp_send_json_succes()
+	 *
+	 * @return void
+	 */
+	static function disable_wp_debug() {
+
+		$wpconfig = ABSPATH . 'wp-config.php';
+
+		$editing_wpconfig = file( $wpconfig );
+
+		file_put_contents( $wpconfig, '' );
+
+		$write_wpconfig = fopen( $wpconfig, 'w' );
+
+		foreach ( $editing_wpconfig as $line ) {
+			if ( false !== strpos( $line, "'WP_DEBUG'" ) || false !== strpos( $line, '"WP_DEBUG"' ) ) {
+				$line = "define( 'WP_DEBUG', false );" . PHP_EOL;
+			}
+			fputs( $write_wpconfig, $line );
+		}
+
+		fclose( $write_wpconfig );
+
+		$response = array(
+			'status'  => 'success',
+			'message' => esc_html__( 'WP_DEBUG was disabled.', 'health-check' ),
+		);
+
+		wp_send_json_success( $response );
+
+	}
+
+	/**
+	 * Enables WP_DEBUG_LOG
+	 *
+	 * @uses copy()
+	 * @uses file()
+	 * @uses file_put_contents()
+	 * @uses fopen()
+	 * @uses strpos()
+	 * @uses fputs()
+	 * @uses fclose()
+	 * @uses wp_send_json_error()
+	 * @uses wp_send_json_succes()
+	 *
+	 * @return void
+	 */
+	static function enable_wp_debug_log() {
```

Comment:
When enabling debug logging, we need to add a rewrite rule to avoid direct access of this file, this is to ensure no private data accidentally ends up in a log that's left publicly available.